Foundation/CGFloat.swift

```diff
@@ -143,6 +143,10 @@ public struct CGFloat {
     
     /// The native value.
     public var native: NativeType
+
+    @_transparent public var double: Double {
+        return Double(native)
+    }
```

Comment:
Can you tell me what you would be happy with here? Potentially brittle casts at all call sites? A function? It can be internal but not also @_transparent. I can then prepare a separate PR and look at merging it into this one.